src/templates/_macros/error_page.html

```diff
@@ -0,0 +1,15 @@
+{% macro error_page(code=_('Error'),
+                    message=_('The page you were looking for was not found.')) %}
+  <section class="main">
+    <div class="subheader">
+      <h1>{{ code }}</h1>
```

Comment:
Can we add an error_messages.js module that maps status codes to messages, and then use that to display the appropriate message here based on the code?